### DISCUSSION TITLE
Download All/ Multiple downloads

### DISCUSSION THREAD

**Discussion by ghost** [LeoKlaus/plappa#174] [Feature Requests]
It would be great to have the ability to initiate multiple downloads, ex select all or select a number to titles to the download cue.

**Comment by LeoKlaus** [comment]
Thank you for the suggestion! This has been requested a number of times now, it's definitely on my to-do list!

**Comment by LeoKlaus** [comment]
Quick update: I've just released the 1.5.0 update for plappa that includes the option to download entire collections at once.

As this is not exactly what you asked for, I will keep this open but I think it might still be interesting for you :)
